components/ws-manager-bridge/src/bridge.ts

```diff
@@ -346,6 +346,15 @@ export class WorkspaceManagerBridge implements Disposable {
                     }
 
                     instance.status.phase = "running";
+                    // let's check if the state is inconsistent and be loud if it is.
+                    if (instance.stoppedTime || instance.stoppingTime) {
```

Comment:
I didn't find any places where either `stoppedTime` or `stoppingTime` would be set without the status being set accordingly. 

OTOH there is a handling of wsman events where the RUNNING phase is being persisted blindly  https://github.com/gitpod-io/gitpod/blob/24f7b609bf79b247d72d1841282639726bcd5891/components/ws-manager-bridge/src/bridge.ts#L158

That said, I'm fine this adding this check here, but it seems, it should be done before the RUNNING phase is persisted bogusly.

cc. @geropl because he reimplemented the status mapping for prebuilds, which seems to be a better pattern for the whole update cascade of states.